pyproject.toml

```diff
@@ -89,6 +89,10 @@ vlm = [
     "torchvision",
     "num2words==0.5.14"
 ]
+latex_math = [
+    "latex2sympy2_extended>=1.0.6",
```

Comment:
```suggestion
```

it's a dependency of math-verify, I think you can remove this. See https://github.com/huggingface/Math-Verify/blob/5d148cfaaf99214c2e4ffb4bc497ab042c592a7a/pyproject.toml#L23